src/shared/prosemirror-plugins/image-upload.ts

```diff
@@ -286,6 +310,41 @@ export class ImageUploader extends PluginInterfaceView<
         event.stopPropagation();
     }
 
+    getAcceptedFileTypesString(types: string[]): string {
+        if (types?.length === 0) {
+            return "";
+        }
+
+        let uploadCaptionString = types[0].replace(/image\//g, "");
+        if (types.length > 1) {
+            uploadCaptionString = types
+                .map((type, i) => {
+                    return i === types.length - 1
```

Comment:
I personally find nested ternary operators far more difficult to read than nested ifs. I would probably just write `if`s in here even if it's a bit more verbose.